tests/test_accelerator.py

```diff
@@ -55,6 +55,25 @@ def parameterized_custom_name_func(func, param_num, param):
 
 
 class AcceleratorTester(AccelerateTestCase):
+    # Should be removed after 1.0.0 release
+    def test_deprecated_values(self):
```

Comment:
How about adding these tests:

- Don't pass any of these deprecated arguments
- Pass only some of these arguments
- Pass some of these arguments but using their defaults